examples/tfengine/generated/org_foundation/devops/main.tf

```diff
@@ -33,13 +33,16 @@ terraform {
 # Create the project, enable APIs, and create the deletion lien, if specified.
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 10.0.1"
+  version = "~> 10.0.2"
 
-  name                    = "example-devops"
-  org_id                  = "12345678"
-  billing_account         = "000-000-000"
-  lien                    = true
+  name            = "example-devops"
+  org_id          = "12345678"
+  billing_account = "000-000-000"
+  lien            = true
+  # Create and keep default service accounts when certain APIs are enabled.
   default_service_account = "keep"
```

Comment:
how does this differ to default_service_account? Do we need both?

Reply:
We do need both. `default_service_account` controls the behaviour for the real default SAs, such as the default GCE SA, GAE SA, etc.

`create_project_sa` controls whether or not to create an additional service account called "project-service-account".  It is just an additional thing that the Project Factory module does after creating the project. But in reality, we don't use this SA. This new flag was recently introduced to address https://github.com/terraform-google-modules/terraform-google-project-factory/issues/526.

Reply:
Ah, got it. Thanks!

Reply:
No problem!